### ISSUE TITLE
Flyout doesn't have aria-role set as dialog

### ISSUE THREAD

**Issue by prateekchandan** [microsoft/fast#3439]
**Describe the bug; what happened?**
Flyout component doesn't have aria-role="dialog" set but it has role="dialog" set. With the aria role missing, the screen readers doesn't apply all dialog accessibility property and thus leading to narrator focus going out of flyouts etc.

**What are the steps to reproduce the issue?**
To reproduce, create any dialog with few focusable elements and then do narrator navigation

Example:

1. In edge, open PDF, click on any of drop down button (highlight/ink)
2. Open narrator
3. Start navigation (Caps + right/left)
4. Try to go out of flyout. 

**What behavior did you expect?**, while it doesn't
The focus should cycle inside the flyout

**In what environment did you see the issue?**

All devices in edge

**Is there any additional context?**
Just adding aria-role="dialog" should fix it

**Comment by chrisdholt** [comment]
I think there is a misunderstanding perhaps here. The correct attribute is “role” for setting an aria role. It sounds like you are expecting modal behavior, but not all flyouts are modal, and many flyouts are non-modal. Can you perhaps elaborate on your expectations? Ultimately the current implementation with “role” is correct for a dialog.

**Comment by chrisdholt** [comment]
closed by #3517 as we do not have this control as a WC
